packages/@okta/vuepress-site/docs/guides/customize-tokens-groups-claim/add-groups-claim-custom-as/index.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Add a Groups claim for a Custom Authorization Server
+---
+
+Use these steps to add a Groups claim to ID tokens and access tokens to perform authentication an authorization using a Custom Authorization Server. See [Authorization Servers](/docs/guides/customize-authz-server/overview/) for more information on the types of authorization servers available to you and what you can use them for.
+
+> **Note:** In this example, the user signing in to your app is assigned to a group called **Midwest Sales**. 
+
+1. In the Admin Console, from the **Security** menu, select **API**, and then select the Custom Authorization Server that you want to configure.
+2. Navigate to the **Claims** tab and click **Add Claim**.
+3. Enter a name for the claim. For this example, name it **GroupsClaim**.
+4. In the **Include in token type** section, leave **Access Token** selected. We are adding the Groups claim to an access token in this example.
+5. Leave **Expression** as the **Value type**.
+6. Enter the following expression as the **Value**: `Groups.startsWith("OKTA", "Midwest", 10)`
```

Comment:
In our blog posts, we typically name the claim `groups` and use the Groups filter with a Regex of `.*`. 

Example: https://developer.okta.com/blog/2019/06/20/spring-preauthorize#activate-groups-claim-on-okta

You might notice that the example doesn't require passing in a `groups` scope either. We just use standard OIDC scopes (`openid`, `email`, and `profile`).

Reply:
@mraible - I can change the claim name to `groups` and use the `Groups` filter with a Regex of `.*`. Just tested that and works fine for me with my example scenario. As you can see, I don't ask them in this process to add a `groups` scope here, nor do I use it in the examples just below - just the `openid` scope. Would you like me to add the other two scopes to my example here - `email` and `profile`??

Reply:
No, I don't think you need to add the other scopes. I just saw `groups` being used in another page that's part of this PR.